src/sentry/seer/similarity/utils.py

```diff
@@ -283,8 +274,7 @@ def _process_frames(frames: list[dict[str, Any]]) -> list[str]:
             and not app_hash
         ):
             raise TooManyOnlySystemFramesException
-        if has_no_filename_or_module:
-            raise NoFilenameOrModuleException
```

Comment:
Since we don't raise the exception, we will start calling seer for stack traces where `filename` and `module` are missing.